CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### AM2R
 
-- Changed: All ammo tanks are now consistently being referred to as "Tanks" rather than "Expansions".
+- Changed: All ammo tanks are now consistently being referred to as "Tanks" rather than "Expansions". If you wish to continue older Multiworld sessions, you will need to re-export and delete the save file.
```

Comment:
What would be necessary to keep old save files compatible?

Reply:
thinking about it, just a quick change, lemme do that rq.